context/app/static/js/components/cells/DatasetsSelectedByExpression/DatasetsSelectedByExpression.jsx

```diff
@@ -140,35 +122,41 @@ function DatasetsSelectedByExpression({
               getContentAnchorEl: null,
             },
           }}
+          helperText="Query measurement refers to Gene Expression (RNA) or DNA Accessibility (ATAC)."
         >
-          <MenuItem value="rna">rna</MenuItem>
-          <MenuItem value="atac">atac</MenuItem>
+          <MenuItem value="rna">Gene Expression (RNA)</MenuItem>
+          <MenuItem value="atac">DNA Accessibility (ATAC)</MenuItem>
         </StyledTextField>
       )}
-      <br />
-      <FormLabel id="min-expression-label">{`Minimum ${queryType} expression`}</FormLabel>
-      <LogSliderWrapper
-        value={minExpressionLog}
-        minLog={-4}
-        maxLog={5}
-        setter={setMinExpressionLog}
-        labelledby="min-expression-label"
-      />
-      <FormLabel id="min-cell-percentage-label">Minimum cell percentage</FormLabel>
-      <SliderWrapper
-        value={minCellPercentage}
-        min={0}
-        max={10}
-        marks={[0, 1, 2, 5, 10]}
-        setter={setMinCellPercentage}
-        labelledby="min-cell-percentage-label"
-      />
-      <br />
-      <Button onClick={handleSubmit} disabled={cellVariableNames.length === 0} variant="contained" color="primary">
-        Run Query
-      </Button>
-      <br />
-      {message}
+      <div>
+        <LogSlider
+          label="Minimum Expression Level"
+          helperText="Set the minimum gene expression level to refine your dataset selections."
+          value={minExpressionLog}
+          minLog={-4}
+          maxLog={5}
+          onChange={(e, val) => setMinExpressionLog(val)}
+          id="min-expression"
+        />
+      </div>
+      <div>
+        <MarkedSlider
+          label="Minimum Cell Percentage (%)"
+          helperText="Set the minimum cell percentage for cells in the datasets to represent the minimum expression level."
```

Comment:
```suggestion
          helperText="Set the minimum cell percentage that must be above the minimum expression level for the dataset to be returned."
```

I have trouble understanding the current text... but I could also be confused about what this actually does.

Reply:
Let's leave the original text for this one until there's user input on this.